cmd/minishift/cmd/root.go

```diff
@@ -45,22 +48,23 @@ import (
 	"github.com/spf13/viper"
 )
 
-var dirs = [...]string{
-	constants.Minipath,
-	constants.MakeMiniPath("certs"),
-	constants.MakeMiniPath("machines"),
-	constants.MakeMiniPath("cache"),
-	constants.MakeMiniPath("cache", "iso"),
-	constants.MakeMiniPath("cache", "oc"),
-	constants.MakeMiniPath("cache", "images"),
-	constants.MakeMiniPath("config"),
-	constants.MakeMiniPath("addons"),
-	constants.MakeMiniPath("logs"),
-	constants.MakeMiniPath("tmp"),
+type dirs struct {
+	Home       string
+	Config     string
+	Machines   string
+	Certs      string
+	Cache      string
```

Comment:
CacheBase

Reply:
@hferentschik I do not agree. These are actual directory names which is consistent with other names. `CacheBase` does not sound consistent.

Reply:
The actual caches live in subfolders. It is a cache root folder...